packages/symantec_edr_cloud/manifest.yml

```
@@ -1,10 +1,10 @@
 format_version: 3.0.2
 name: symantec_edr_cloud
-title: Symantec EDR Cloud
-version: "1.4.0"
+title: Symantec EDR Cloud (deprecated)
```

For consistency with other packages, I suggest "(Deprecated)".
```
% yq .title packages/*/manifest.yml | grep -i deprecated
Blue Coat Director Logs (Deprecated)
F5 Logs (Deprecated)
Juniper JunOS (Deprecated)
Juniper NetScreen (Deprecated)
Arbor Peakflow SP Logs (Deprecated)
Radware DefensePro Logs (Deprecated)
```

-----

```suggestion
title: Symantec EDR Cloud (Deprecated)
```

---

I was going from a majority rule which is dominated by data streams, these use lower case.

Changed.